sources/models/src/lib.rs

```diff
@@ -52,6 +52,11 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-dev/mod.rs)
```

Comment:
Can we also update the aws-dev header to say "AWS development build" to clarify the distinction?  It won't let me make a suggestion.

Same in `variants/README.md` - the headers are currently the same there, should probably make it match this wording.  And in that file, there are clarifications for VMWare that didn't make it back into the aws-dev paragraph.  I think it'd be cleaner to have each describe its setup, instead of only saying the differences in the VMWare entry.